app/server.js

```diff
@@ -16,19 +15,9 @@ Object.keys(config).map((configKey) => {
 // Routes
 const app = new App().createApp();
 app.use('/', routes);
-
-// Run application on configured port
 if (config.env === 'development') {
-  app.listen(config.port, () => {
-    browserSync({
-      files: ['./views/**/*.*', '../public/**/*.*'],
-      notify: true,
-      open: false,
-      port: config.port,
-      proxy: `localhost:${config.port}`,
-      ui: false,
-    });
-  });
+  logger.info(` Marketing Pages - \x1b[35m${config.appBaseUri}\x1b[0m`);
```

Comment:
What the ???

Reply:
makes it pink 😄

Reply:
and a clickable link for the lazy ones

Reply:
ie me

Reply:
ooh fancy. OK mmm in that case can we for marketing pages make it navigate to `{config.appBaseUri}/supplier/solution/100000-001` then it'll be a clickable link that works 😂

Reply:
YES! 🎉